dev/tasks/linux-packages/travis.linux.arm64.yml

```diff
@@ -15,16 +15,17 @@
 # specific language governing permissions and limitations
 # under the License.
 
+arch: arm64-graviton2
+virt: vm
 os: linux
-arch: arm64
-dist: bionic
+dist: focal
+group: edge
 language: minimal
 
 addons:
   apt:
     packages:
       - apt-utils
-      - createrepo
```

Comment:
We need `createrepo` or `createrepo_c` for testing RPM packages.
`createrepo` exists only on Ubuntu 16.04 and 18.04.
`createrepo_c` exists only on Ubuntu 20.10 or later.

https://packages.ubuntu.com/search?keywords=createrepo

Can we use `bionic` with `arm64-graviton2`?

Reply:
Tried with bionic as well, but apt was unable to locate `createrepo`. Shall I try with `createrepo_c` on focal?